playwright-e2e/tests/page-formating-tests.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { Page, expect, test } from '@playwright/test';
+
+import { urlConfig } from '../utils/urls';
+
+test.describe('Start application details e2e journey in English', () => {
+  let page: Page;
+  test.beforeEach(async ({ browser }) => {
+    const context = await browser.newContext({
+      locale: 'en',
+    });
+    page = await context.newPage();
+    await page.goto(urlConfig.citizenStartUrl);
+  });
+  test('Start up jourey url is equal to the initial url', async () => {
```

Comment:
typo: journey

Reply:
this test might be not needed as it could be incorporated into the other tests and reduce number of tests being run